cscs-checks/apps/amber/amber_check.py

```diff
@@ -58,102 +59,55 @@
             'num_switches': 1
         }
     }
     maintainers = ['VH', 'SO']
-    tags = {'scs', 'external-resources'}
-
-    benchmark = parameter([
-        # NVE simulations
-        'Cellulose_production_NVE',
-        'FactorIX_production_NVE',
-        'JAC_production_NVE_4fs',
-        'JAC_production_NVE',
-    ])
-
-    @run_after('init')
-    def download_files(self):
-        self.prerun_cmds = [
-            # cannot use wget because it is not installed on eiger
-            f'curl -LJO https://github.com/victorusu/amber_benchmark_suite'
-            f'/raw/main/amber_16_benchmark_suite/PME/{self.benchmark}.tar.bz2',
-            f'tar xf {self.benchmark}.tar.bz2'
-        ]
-
-    @run_after('init')
-    def set_energy_and_tolerance_reference(self):
-        self.ener_ref = {
-            # every system has a different reference energy and drift
-            'Cellulose_production_NVE': (-443246, 5.0E-05),
-            'FactorIX_production_NVE': (-234188, 1.0E-04),
-            'JAC_production_NVE_4fs': (-44810, 1.0E-03),
-            'JAC_production_NVE': (-58138, 5.0E-04),
-        }
-
-    @run_after('setup')
-    def set_executable_opts(self):
-        self.executable_opts = ['-O',
-                                '-i', self.input_file,
-                                '-o', self.output_file]
-        self.keep_files = [self.output_file]
-
-    @run_after('setup')
-    def set_sanity_patterns(self):
-        energy = sn.extractsingle(r' Etot\s+=\s+(?P<energy>\S+)',
-                                  self.output_file, 'energy', float, item=-2)
-        energy_reference = self.ener_ref[self.benchmark][0]
-        energy_diff = sn.abs(energy - energy_reference)
-        ref_ener_diff = sn.abs(self.ener_ref[self.benchmark][0] *
-                               self.ener_ref[self.benchmark][1])
-        self.sanity_patterns = sn.all([
-            sn.assert_found(r'Final Performance Info:', self.output_file),
-            sn.assert_lt(energy_diff, ref_ener_diff)
-        ])
-
-    @run_after('setup')
-    def set_generic_perf_references(self):
-        self.reference.update({'*': {
-            self.benchmark: (0, None, None, 'ns/day')
-        }})
-
-    @run_after('setup')
-    def set_perf_patterns(self):
-        self.perf_patterns = {
-            self.benchmark: sn.extractsingle(r'ns/day =\s+(?P<perf>\S+)',
-                                             self.output_file, 'perf',
-                                             float, item=1)
-        }
 
 
 @rfm.simple_test
-class AmberGPUCheck(AmberBaseCheck):
+class amber_gpu_check(AmberCheck):
+    valid_systems = ['daint:gpu', 'dom:gpu']
     num_tasks = 1
-    num_tasks_per_node = 1
     num_gpus_per_node = 1
-    valid_systems = ['daint:gpu', 'dom:gpu']
-    executable = 'pmemd.cuda.MPI'
-    input_file = 'mdin.GPU'
-    output_file = 'amber.out'
+    num_tasks_per_node = 1
     descr = f'Amber GPU check'
     tags = {'maintenance', 'production', 'health'}
```

Comment:
I would vote for adding some generic tags at the library level, too, and here we just augment them (or replace them if we like). What about `sciapp`, `chemistry` ?